### ISSUE TITLE
Add .doc NEWS type

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4224]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4224|
|Type|enhancement|
|Created|2010-01-15 21:05:48Z|
|Branch|https://github.com/twisted/twisted/tree/doc-news-4224-2|

Currently 95% of the documentation tickets are marked as .misc changes, but might make sense to have a .doc type file instead.

<details><summary>Searchable metadata</summary>

```
trac-id__4224 4224
type__enhancement enhancement
reporter__thijs thijs
priority__normal normal
milestone__totally_automated_release_infrastructure totally automated release infrastructure
branch__branches_doc_news_4224_2 branches/doc-news-4224-2
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1263589548000000 1263589548000000
changetime__1274822338000000 1274822338000000
version__None None
owner__ 
cc__thijs cc__khorn cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn commented|
|-|-|

+100

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Yea, I think this is a good idea.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [29051]) Branching to 'doc-news-4224'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [29052]) Support .doc files

refs [#4224](https://github.com/twisted/twisted/issues/4224)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun removed owner|
|-|-|

The tool supports .doc files in the branch.  Suggestions for better presentation welcome.

Once this is accepted, we need to update the wiki page that talks about news fragments to mention this new type and give some guidelines for what to put in .doc files.  We'll also need to update `enforcenews.py` to recognize this type.

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|ivank commented|
|-|-|

1. Please remove the unused local `basename = ticket.basename()`

Everything else looks good. I am happy with the current presentation.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

(In [29058]) Branching to 'doc-news-4224-2'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set status to `closed`|
|-|-|

(In [29060]) Merge doc-news-4224-2

Author: exarkun
Reviewer: ivank
Fixes: [#4224](https://github.com/twisted/twisted/issues/4224)

Add support for a new kind of news fragment, .doc.  These are aggregated
into a new section of the news file by the news builder.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
